app/web_hub.go

```diff
@@ -203,7 +203,6 @@ func (a *App) PublishSkipClusterSend(message *model.WebSocketEvent) {
 
 func (a *App) InvalidateCacheForChannel(channel *model.Channel) {
 	a.Srv.Store.Channel().InvalidateChannel(channel.Id)
-	a.InvalidateCacheForChannelByNameSkipClusterSend(channel.TeamId, channel.Name)
```

Comment:
You should be using here the `InvalidateCacheForChannelByName` function.

Reply:
so call both ```InvalidateChannel``` and ```a.Srv.Store.Channel().InvalidateChannelByName(teamId, name)``` ?

Reply:
Yes, and no. My proposal is to use the `InvalidateCacheForChannelByName` method name from the `App` layer, but you can just move the code right here and remove the entire `InvalidateCacheForChannelByName` method. The `InvalidateChannel` only invalidate the channel cache by Id, not the channel cache by name.

Reply:
@jespino is this the change you meant?